cpp/benchmarks/string/convert_fixed_point_benchmark.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <fixture/benchmark_fixture.hpp>
+#include <synchronization/synchronization.hpp>
+
+#include <benchmark/benchmark.h>
+#include <benchmarks/common/generate_benchmark_input.hpp>
+
+#include <cudf/strings/convert/convert_fixed_point.hpp>
+#include <cudf/strings/convert/convert_floats.hpp>
+#include <cudf/types.hpp>
+
+#include <cudf_test/base_fixture.hpp>
+
+namespace {
+
+std::unique_ptr<cudf::column> get_strings_column(cudf::size_type rows)
+{
+  std::unique_ptr<cudf::table> result =
+    create_random_table({cudf::type_id::FLOAT32}, 1, row_count{static_cast<cudf::size_type>(rows)});
+  return cudf::strings::from_floats(result->release().front()->view());
+}
+
+}  // anonymous namespace
+
+class StringsToFixedPoint : public cudf::benchmark {
+};
+
+template <typename fixed_point_type>
+void convert_to_fixed_point(benchmark::State& state)
+{
+  const auto rows         = static_cast<cudf::size_type>(state.range(0));
+  const auto strings_col  = get_strings_column(rows);
+  const auto strings_view = cudf::strings_column_view(strings_col->view());
+  const auto dtype = cudf::data_type{cudf::type_to_id<fixed_point_type>(), numeric::scale_type{-2}};
+
+  for (auto _ : state) {
+    cuda_event_timer raii(state, true);
+    volatile auto results = cudf::strings::to_fixed_point(strings_view, dtype);
+  }
+
+  using rep_type = typename cudf::device_storage_type_t<fixed_point_type>;
+  // bytes_processed = bytes_input + bytes_output
+  state.SetBytesProcessed(state.iterations() *
+                          (strings_view.chars_size() + rows * sizeof(rep_type)));
```

Comment:
I think you can use `cudf::size_of` here.
```suggestion
  // bytes_processed = bytes_input + bytes_output
  state.SetBytesProcessed(state.iterations() *
                          (strings_view.chars_size() + rows * cudf::size_of(fixed_point_type)));
```